src/background.js

```diff
@@ -338,8 +336,11 @@ const rollout = {
   async enterprisePolicyCheck(event, results) {
     results.evaluateReason = event;
 
-    // Reset skipHeuristicsCheck
-    await rollout.setSetting(DOH_SKIP_HEURISTICS_PREF, false);
+    // Check if trrModePrefHasUserValue determined to not enable add-on on first run
+    let skipHeuristicsCheck = await rollout.getSetting(DOH_SKIP_HEURISTICS_PREF, false);
+    if (skipHeuristicsCheck) {
+      return;
+    }
```

Comment:
This quits the enterprisePolicyCheck check if we've already detected that the user had trrModePrefHasUserValue